sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/QuickPulseDataCollector.java

```diff
@@ -96,6 +95,33 @@ synchronized FinalCounters peek() {
         return null;
     }
 
+
+    synchronized void addOtelMetric(TelemetryItem telemetryItem){
+        if (!isEnabled()) {
+            // quick pulse is not enabled or quick pulse data sender is not enabled
+            return;
+        }
+
+        if (!telemetryItem.getInstrumentationKey().equals(instrumentationKeySupplier.get())) {
+            return;
+        }
+
+        Float sampleRate = telemetryItem.getSampleRate();
```

Comment:
Please remove unused code (sample rate, item count)

Reply:
removed itemCount as it was unused